hydrabus_framework/modules/base.py

```diff
@@ -1,10 +1,11 @@
-from core.helper import Helper
```

Comment:
You should never have to class in the same file. Moreover, your are missleading the aim of an Abstract class.
For example, it should not have any name, description and so forth

Reply:
Done [#8df850f](https://github.com/hydrabus-framework/framework/commit/8df850f2a9ecd72323d208ef8be2b72a621336ae) and [#10c3681](https://github.com/hydrabus-framework/framework/commit/10c368189272dfb79bbaec95279da987552428a0)

Reply:
The ffile should named the same than the module :)

Reply:
Done [#7cc9a18](https://github.com/hydrabus-framework/framework/commit/7cc9a1807688ca416ecd5600ee9ff0c410496a68)